pom.xml

```diff
@@ -456,7 +456,7 @@
         </repository>
     </repositories>
     <properties>
-        <swagger-parser-version>1.0.9-SNAPSHOT</swagger-parser-version>
+        <swagger-parser-version>1.0.9</swagger-parser-version>
```

Comment:
I have no idea why that snapshot was deleted--I've redeployed it so 1.0.9 should now be available.